astro/src/content/docs/lifecycle/authenticate-users/multi-factor-authentication.mdx

```diff
@@ -427,8 +428,8 @@ user_two_factor_removed=`echo $user| jq 'del(.[].twoFactor[])' -`
 curl -XPUT -H 'Content-type: application/json' -H "Authorization: $API_KEY" 'https://sandbox.fusionauth.io/api/user/00000000-0000-0000-0000-000000000004' -d "$user_two_factor_removed"
 ```
 
-The reason you need to retrieve the user and modify the data, then use `PUT` to update it, is because of how `PATCH` handles arrays.
-[Read this tracking issue for more info.](https://github.com/FusionAuth/fusionauth-issues/issues/441)
+{/* TODO: check if this still makes sense */}
+<StaticPatchNote />
```

Comment:
I would not put the static patch note here. I do think that you can remove `Read this tracking issue for more info.](https://github.com/FusionAuth/fusionauth-issues/issues/441)` and point folks at https://fusionauth.io/docs/apis/#the-patch-http-method